common/web/gesture-recognizer/src/mouseEventEngine.ts

```diff
@@ -9,20 +9,33 @@ namespace com.keyman.osk {
 
     private hasActiveClick: boolean = false;
     private disabledSafeBounds: number = 0;
-    private ignoreSequence: boolean = false;
+
+    private IDENTIFIER_SEED: number;
```

Comment:
So the identifiers are unique only within a given instance of the class. Is that sufficient?

Or should we be doing:

```suggestion
    private static IDENTIFIER_SEED: number;
```